integration/integrate-with-image-registries.adoc

```diff
@@ -10,7 +10,7 @@ toc::[]
 
 When you integrate with image registries, you can view important image details, such as image creation date and Dockerfile details (including image layers).
 
-Once you have integrated with your registry, scan images, view image components, and apply security policies either before you deploy images or for the already deployed images in your clusters.
+Once you have integrated with your registry, you can scan images, view image components, and apply security policies either before you deploy images or for the already deployed images in your clusters.
```

Comment:
s/Once/After/ // ISG around p374.

Also, maybe "After you integrate with your registry, you can..." Maybe "integrate {product-name-attr}".  Up to you.

The tail end of that sentence seems a little strained.  What about:

"...and apply security policies before you deploy images or to existing images that are already deployed in your clusters."  That might not be better, it's "already deployed images" that caught my eye.

Also, is it really "or" or is it really "and"?  Just checking.